pkg/node/kubelet.go

```diff
@@ -84,3 +87,74 @@ func StartKubeProxy(cfg *config.MicroshiftConfig) error {
 
 	return nil
 }
+
+const (
+	// proxy component name
+	component = "kubeproxy"
+)
+
+type ProxyOptions struct {
+	options *kubeproxy.Options
+}
+
+func NewKubeProxyServer(cfg *config.MicroshiftConfig) *ProxyOptions {
+	s := &ProxyOptions{}
+	s.configure(cfg)
+	return s
+}
+
+func (s *ProxyOptions) Name() string           { return component }
+func (s *ProxyOptions) Dependencies() []string { return []string{"kube-apiserver"} }
+
+func (s *ProxyOptions) configure(cfg *config.MicroshiftConfig) error {
+	if err := config.KubeProxyConfig(cfg); err != nil {
+		logrus.Infof("Failed to create a new kube-proxy configuration: %v", err)
+		return err
+	}
+	args := []string{
+		"--logtostderr=" + strconv.FormatBool(cfg.LogDir == "" || cfg.LogAlsotostderr),
```

Comment:
@rootfs & @fzdarsky Do we need to add `--congfig` in `args`?  `kubeProxy.Options` has a mechanism to read a config file using 
```
opts := kubeproxy.NewOptions()
opts.ConfigFile = cfg.DataDir + "/resources/kube-proxy/config/config.yaml"
opts.Complete()
```
`opts.Complete()` loads configuration from a file.
Right now I am not injecting `KubeProxyConfiguration` using `--config` flag and it works.

Reply:
I'm totally fine doing that.